x-pack/plugins/security/server/authentication/providers/kerberos.ts

```diff
@@ -107,7 +107,7 @@ export class KerberosAuthenticationProvider extends BaseAuthenticationProvider {
       return DeauthenticationResult.failed(err);
     }
 
-    return DeauthenticationResult.redirectTo('/logged_out');
+    return DeauthenticationResult.redirectTo(`${this.options.basePath.get(request)}/logged_out`);
```

Comment:
This will redirect us to the "space relative" basePath, which then causes https://github.com/elastic/kibana/blob/2d1c239fddb299a80c4e051e85676ecc66511a68/x-pack/legacy/plugins/security/public/views/logged_out/logged_out.tsx#L33 to take us to the "space relative" `/`. Should we use the server root base path here instead?

Reply:
🙈I of all people should have seen that, thanks for catching this! I modeled it off of the [OIDC implementation](https://github.com/elastic/kibana/blob/fce52133d8818925aec9d5807ebbbd06693fcf38/x-pack/plugins/security/server/authentication/providers/oidc.ts#L434) without much thought.

We absolutely should use the server root base path here, and in the rest of the auth providers instead.